### ISSUE TITLE
Fountain code blocks in markdown files don't render

### ISSUE THREAD

**Issue by Amparose** [GamerGirlandCo/obsidian-fountain-revived#10]
Installed via BRAT, disabled the [original plugin](https://github.com/Darakah/obsidian-fountain), restarted Obsidian, and the following doesn't render:

```
```fountain
INT.

In a dusty room

WRITER 1
Why doesn't this plugin work?

MONITOR
...

CUT TO:

EXT.

WRITIER 1
Sigh
\```
```

Backslash at end is just to show backticks on Github. It works with the original plugin. Is there a different workflow with this plugin? There's no documentation past installation so assumed it was same.

**Comment by wesleyboar** [comment]
I did not test that. It may not work. But I know what does work — opening `.fountain` files.

1. Create new file (perhaps outside of Obsidian) called `test.fountain`.
2. Open the file with Obsidian.
3. Add the fountain syntax content:

	```
	INT.

	In a dusty room

	WRITER 1
	Why doesn't this plugin work?

	MONITOR
	...

	CUT TO:

	EXT.

	WRITIER 1
	Sigh
	```

This plugin has very different code than the old one, which I suspect is so it can support [Obsidian "Live Preview"](https://help.obsidian.md/Live+preview+update).

**Comment by wesleyboar** [comment]
There is a video in issue #1 that shows the plugin working (albeit with the problem described in #1).

So, your title seems scoped to your assumption of how it works. Consider a title like:

> Doesn't highlight fountain syntax in markdown code block

**Comment by Amparose** [comment]
You're correct that coming from the old plugin—seeing it referenced in the readme and the title "revived" encourages interpretation that this is a fork—that I assumed this would work as part of Obsidian's markdown files rather than adding support for Fountain files. Would help if this was stated more clearly as my issue may not be a bug at all but an out-of-scope feature request.

Will test this again tomorrow and see if this can run alongside the old plugin that does support Fountain code blocks that show in Live Preview, albeit as a rendered code-block.

**Comment by wesleyboar** [comment]
Yes, "revived" is misleading. Had I used that feature on the first one, I'd have done the same as you.

**Comment by Amparose** [comment]
> This plugin has very different code than the old one, which I suspect is so it can support [Obsidian "Live Preview"](https://help.obsidian.md/Live+preview+update).

Well, unfortunately, the two plugins don't play nice together, which is a shame as it would be useful to be able to view .fountain files and include fountain text in .md notes. Playing with just this plugin, it would be useful it added a function to create .fountain files in Obsidian.

But... before that the bigger issue is how [the formatting is janky](https://github.com/GamerGirlandCo/obsidian-fountain-revived/issues/11).

**Comment by wesleyboar** [comment]
Yes. Indeed. I am most excited about trying to fix [styling] via [my fork](https://github.com/wesleyboar/obsidian-fountain-revived).

I have a branch of code ([minimal work so far](https://github.com/wesleyboar/obsidian-fountain-revived/compare/dev...wesleyboar:obsidian-fountain-revived:chore/clean-up-styling)) where I will work on that.

Do you care to make a new issue with those screenshots?

**Comment by wesleyboar** [comment]
(I believe this plugin was a proof-of-concept project.)

**Comment by Amparose** [comment]
> Yes. Indeed. I am most excited about trying to fix [styling] via [my fork](https://github.com/wesleyboar/obsidian-fountain-revived).
> 
> I have a branch of code ([minimal work so far](https://github.com/wesleyboar/obsidian-fountain-revived/compare/dev...wesleyboar:obsidian-fountain-revived:chore/clean-up-styling)) where I will work on that.
> 
> Do you care to make a new issue with those screenshots?

Done and linked above. Oooo! I'll check out your fork when I next get a moment as I decided to hack on the original plugin to make better fenced fountain blocks.

**Comment by wesleyboar** [comment]
Well, maybe more to see in the future. All I've done so far is fix and improve development environment, and add a fallback font.

**Comment by wesleyboar** [comment]
I abandoned my fork in favor of https://github.com/chuangcaleb/obsidian-fountain-editor.
